### DISCUSSION TITLE
How to connect with NX-8?

### DISCUSSION THREAD

**Discussion by chrodriguez** [tjko/nxgipd#23] [General]
I've a NX-8 alarm, not NX-8E. Does anyone write some guide to understand how to connect a rpi with this board?

Thanks in advance

**Comment by tjko** [comment]
You will need following:
- NX-584 interface module
- USB to RS-232 adapter (something like: https://www.amazon.com/gp/product/B0769DVQM1/)
- optionally you may need RS-232 extension cable if RPi is not in close proximity...

You can install it yourself if you know the (master) programming code for your panel. Otherwise you may need to get your alarm provider install it for you.  Just make sure interface module is configured as suggested in the README file...
